sources/osgShadow/ShadowTextureAtlas.js

```diff
@@ -39,15 +38,11 @@ ShadowTextureAtlas.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectI
         return new ShadowTextureAtlas();
     },
 
-    setLightNumber: function ( value ) {
-        this._lightNumber = value;
+    getLightsIndexes: function () {
```

Comment:
it's not really an indexes, it's an lightNumberArray, it's confusing to read index from other classes.